File: Tasks/Java/List.java
package todolist;

import java.util.*;

class List{
	
	ArrayList<Task> list;
	
	List(){
		this.list = new ArrayList<Task>();
	}
	
	public void addTask(Task t) {
		this.list.add(t);
	}
	
	public void removeTask() {
		this.list.remove(this.list.size() - 1);
	}
	
	public void printTask() {
		 for(int i = 0; i < this.list.size(); i++) {
			 System.out.printf("%d. %s\n",i+1, this.list.get(i).getName());
	}
	
	
	
	
	}	
}

File: Tasks/Python/List.py
from Task import Task

class List:
	def __init__(self):
		self.list =  []
		
	def addTasks(self, Task):
		self.list.append(Task);
		
	def removeTask(self):
		del self.list[-1]
				
	def printTask(self):
		 print "*************To Do List*********************"
		 for x in range(len(self.list)):
			print self.list[x].getName() 